cypress/e2e/10-settings-log-streaming.cy.ts

```diff
@@ -104,6 +104,7 @@ describe('Log Streaming Settings', () => {
 			.eq(0)
 			.find('.el-dropdown-selfdefine')
 			.click();
+		cy.wait(100);
```

Comment:
can we try to fix this without timeout? as a lot of these would slow down the test suite too much... like waiting for element to show up...

Reply:
removed. works without as well, the main issue was the wrong destination id